cxx4opencl/intro.txt

```diff
@@ -1,46 +1,63 @@
-// Copyright 2019-2020 The Khronos Group. This work is licensed under a
+// Copyright 2019-2021 The Khronos Group. This work is licensed under a
 // Creative Commons Attribution 4.0 International License; see
 // http://creativecommons.org/licenses/by/4.0/
 
 [[intro]]
 == Introduction
 
-This language is built on top of OpenCL C v2.0 and {cpp}17 enabling
+This language is built on top of OpenCL C 3.0 unified and {cpp}17 enabling
```

Comment:
Well currently C++ for OpenCL 1.0 is built on top of OpenCL 2.0 which is backward compatible with OpenCL 1.X and C++ for OpenCL 2021 is compatible with OpenCL 3. So we do cover all OpenCL C dialects now.

However, at the end of this section, we have a clarification of version compatibility so I don't see a need to have it here especially since it can't be explained properly at this point so it looks a bit confusing in my view.

Reply:
I see the reasoning now. Somehow I confused myself between the concepts of backward and forward compatibility. I agree that current description is good as is.